sdk/communication/Azure.Communication.Rooms/src/Generated/Models/ParticipantProperties.Serialization.cs

```diff
@@ -15,15 +15,8 @@ internal partial class ParticipantProperties : IUtf8JsonSerializable
         void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
         {
             writer.WriteStartObject();
-            if (Role != null)
```

Comment:
Property is declared in specification as non-nullable struct, which is changed in custom code